apps/components-e2e/src/components/slider/slider.e2e.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// tslint:disable no-lifecycle-call no-use-before-declare no-magic-numbers
+// tslint:disable no-any max-file-line-count no-unbound-method use-component-selector
+
+import { waitForAngular } from '../../utils';
+
+fixture('Slider')
+  .page('http://localhost:4200/slider')
+  .beforeEach(async () => waitForAngular());
```

Comment:
Please reset the window width to the default to prevent flakiness if you rely on coordinates or width calculations. Other tests can influence this. To prevent this we are resetting the window for every test run.

```suggestion
  .beforeEach(async () => {
    await resetWindowSizeToDefault();
    await waitForAngular();
  });
```

Reply:
I'll update this, thanks.